website/docs/en/contribute/development/profiling.md

```diff
@@ -6,6 +6,14 @@ Since different profilers have different strengths. It is good to use more than
 
 <!-- toc -->
 
+## Build release version with debug info
+
+Performance analysis should be conducted on a release version that includes debug information. This approach ensures accurate performance results while providing sufficient debug information for analysis. Use the following command to build a release version with debug information:
```

Comment:
Can we tell users how to use the debug release? Like using pnpm link or something else.

Reply:
it turns out link protocol works perfectly, but it only works for pnpm, I don't find a idea way works both on npm & yarn, so I just write pnpm link guide